pkg/publisher/publisher.go

```diff
@@ -93,24 +93,25 @@ func New(ctx context.Context, region string, clusterID string) (Publisher, error
 	// Case 3: Cx blocked IMDS access and not using IRSA (which means region == "") OR
 	// not specified clusterID then its a Cx error
 	if region == "" || clusterID == "" {
```

Comment:
Minor nit around code structure. 

Code is structured as below right now.
`
if region == "" || clusterID == ""
{
   if clusterID == ""{
   }
   
   if region == "" {
   }
}
`
Why do we need the upper level `if region == "" || clusterID == ""`? Instead below will suffice right? Prior to this diff update, we had few additional things we were doing outside `if region == ""` and `if clusterID == ""`blocks but with the below changes, everything is confined to both the individual `if` blocks..

`
 if clusterID == ""{
   }
   
   if region == "" {
   }
`
From what I can see, looks like something we don't really need the upper check.

Reply:
Yes, we dont need it. Have Updated the logic